nodeup/pkg/model/docker.go

```diff
@@ -439,11 +439,12 @@ var dockerVersions = []packageVersion{
 		},
 	},
 
-	// 18.09.9 - CentOS / Rhel7
+	// 18.09.9 - CentOS7 / Rhel7
+	// (relies on container-selinux, so not suitable for amazonlinux2)
```

Comment:
Because 18.09 goes away, or because container-selinux will be supported on amazonlinux2?

If it's the confusing phrasing, re-reading, I definitely agree...  perhaps "(we can't use this package on amazonlinux2, because it relies on container-selinux)"

Reply:
With this fix we create a separate section for Amazon Linux 2. I think it is enough. Maybe add the comment to the Amazon Linux 2 section?
Starting 1.18, we use only the static package and there is no need for extra explanations.

Reply:
Ah gotcha.  I'll add the comment there!

Reply:
Great. Thanks!